src/lib/hooks.js

```diff
@@ -52,6 +52,7 @@ const api = {
 					color,
 					fontColor,
 					iconColor,
+					title,
```

Comment:
Did you forget the custom `offlineTitle`?
The `offlineTitle` is used on the `LeaveAMessage` container.

Reply:
yes :p
I'll do that